lib/web_ui/dev/chrome_installer.dart

```diff
@@ -227,3 +227,63 @@ Future<String> fetchLatestChromeVersion() async {
     client.close();
   }
 }
+
+/// Get the Chrome Driver version for the system Chrome.
+///
+/// It is only used by integration tests for now.
+/// TODO(nurhan): Merge the chrome download code in the unit tests with this
+/// one. Have both of them relying to the same chrome version.
+Future<String> queryChromeDriverVersion() async {
+  final int chromeVersion = await _querySystemChromeVersion();
+  final io.File lockFile = io.File(
+        path.join(environment.webUiRootDir.path, 'dev', 'driver_version.yaml'));
+  YamlMap _configuration = loadYaml(lockFile.readAsStringSync()) as YamlMap;
+  final String chromeDriverVersion =
+      _configuration['chrome'][chromeVersion] as String;
+  return chromeDriverVersion;
+}
+
+Future<int> _querySystemChromeVersion() async {
```

Comment:
nit: recommend renaming to `_querySystemChromeMajorVersion` because this function does not return the full version number, only the major version number.

Reply:
done